test/mysql/ConnectionTest.chpl

```diff
@@ -0,0 +1,85 @@
+use DatabaseCommunicator;
+use DatabaseCommunicator.DatabaseCommunicationObjects.ErrorTypes;
+use DatabaseCommunicator.DatabaseCommunicationObjects.QueryBuilder;
+use UnitTest;
+use MySQL;
+
+proc simpleConnectionTest(test: borrowed Test) throws {
+    var conHandler = new ConnectionHandler(MySQLConnection, "localhost;testdb;root;password");
+    conHandler.close();
+
+    // Invalid connection string, should throw exception:
+    conHandler = new ConnectionHandler(MySQLConnection, "localhost;");
+}
+
+proc configConnectionTest(test: borrowed Test) throws {
+    var conHandler = ConnectionHandlerWithConfig(MySQLConnection, "dbconfig.toml");
+    conHandler.close();
+
+    // different root element name in TOML file
+    conHandler = ConnectionHandlerWithConfig(MySQLConnection, "dbconfig2.toml", "dbconf");
+    conHandler.close();
+
+    // wrong root element in TOML file, should throw exception:
+    conHandler = ConnectionHandlerWithConfig(MySQLConnection, "dbconfig.toml", "root");
+    conHandler.close();
+
+    // Invalid file, should throw exception saying file not found
+    conHandler = ConnectionHandlerWithConfig(MySQLConnection, "nofile");
+
+    // Invalid file (not TOML), should throw TOML parsing error
+    conHandler = ConnectionHandlerWithConfig(MySQLConnection, "TransactionTest.chpl");
+}
+
+proc testAutocommit(test: borrowed Test) throws {
```

Comment:
I would suggest to break this test.

Reply:
Valid point, I too was a bit reluctant to make it this long but then forgot to update it. I'll split it.